index.bs

```diff
@@ -4484,9 +4582,17 @@ When this method is invoked, the [=authenticator=] MUST perform the following pr
     which approach is implemented by the [=authenticator=], by some positive value.
     If the [=authenticator=] does not implement a [=signature counter=], let the [=signature counter=] value remain constant at
     zero.
+1. If |attestationFormats|:
+    <dl class="switch">
+        : is [=list/is not empty|not empty=]
+        :: let |attestationFormat| be the first supported [=attestation statement format=] from |attestationFormats|, taking into account |enterpriseAttestationPossible|. If none are supported, fallthrough to:
```

Comment:
This is covered below too, so is it still needed here?

```suggestion
        :: let |attestationFormat| be the first supported [=attestation statement format=] from |attestationFormats|. If none are supported, fallthrough to:
```

Reply:
(See reply on [previous comment](https://github.com/w3c/webauthn/pull/1663#discussion_r968203551).)